2-D_Array/C++/addition_by_factor.cpp

```diff
@@ -0,0 +1,70 @@
+#include<iostream>
+using namespace std;
+main()
+{
+    int n,m;
+    cout<<"Enter dimensions of 2-D array: ";
+    cin>>n>>m;
+
+    int arr1[n][m],arr2[n][m],i,j;
+    cout<<"Enter factor of first and second array: ";
+
+    int a,b;
+    cin>>a>>b;
+
+    cout<<"Enter values of first array: ";
+    for(i=0;i<n;i++)
+    {
+        for(j=0;j<m;j++)
+        {
+            cin>>arr1[i][j];
+        }
+        cout<<endl;   //to add a new line when elements of other rwo are added
+    }
+
+    cout<<"Enter values of second array: ";
+    for(i=0;i<n;i++)
+    {
+        for(j=0;j<m;j++)
+        {
+            cin>>arr2[i][j];
+        }cout<<endl;
+    }
+
+    int arr[n][m];
+    //array in which answer i.e. a*arr1+b*arr2 is stored
+    for(i=0;i<n;i++)
+    {
+        for(j=0;j<m;j++)
+        {
+            arr[i][j]=a*arr1[i][j]+b*arr2[i][j];
+        }
+    }
+
+    cout<<"Added matrix is: ";
+    for(i=0;i<n;i++)
+    {
+        for(j=0;j<m;j++)
+        {
+            cout<<arr[i][j]<<" ";
+        }
+        cout<<endl;
+    }
+}
+// Enter dimensions of 2-D array: 2 2
```

Comment:
Add a new line above this.

Reply:
should I do all of these changes even in c program